cypress/e2e/smoke/UPM/passwordReset/resetPasswordLoggedOut.cy.js

```diff
@@ -18,30 +18,31 @@ describe('QATEST-173757: Verify user should be able to reset password when logge
         onBeforeLoad(win) {
           win.localStorage.setItem(
             'config.server_url',
-            Cypress.env('configServer')
+            Cypress.env('stdConfigServer')
+          )
+          win.localStorage.setItem(
+            'config.app_id',
+            Cypress.env('stdConfigAppId')
           )
-          win.localStorage.setItem('config.app_id', Cypress.env('configAppId'))
         },
       })
       cy.visit(Cypress.env('derivComProdURL'), {
         onBeforeLoad(win) {
           win.localStorage.setItem(
             'config.server_url',
-            Cypress.env('configServer')
+            Cypress.env('stdConfigServer')
+          )
+          win.localStorage.setItem(
+            'config.app_id',
+            Cypress.env('stdConfigAppId')
           )
-          win.localStorage.setItem('config.app_id', Cypress.env('configAppId'))
         },
       })
       cy.c_visitResponsive(
-        `${Cypress.env('derivComProdURL')}/reset-password`,
+        `${Cypress.env('derivComProdURL')}reset-password`,
         size
       )
-      cy.findByRole(
-        'button',
-        { name: 'whatsapp icon' },
-        { timeout: 30000 }
-      ).should('be.visible')
-      cy.findByPlaceholderText('example@email.com').clear().type(userEmail)
+      cy.get('input#email-reset-password').clear().type(userEmail)
```

Comment:
suggestion: Can you try
```suggestion
      cy.findByLabelText('Email').clear().type(userEmail)
```

Reply:
I've tried this but I think it does not work as it is a `class="floating-label"`